utils/data/timeseries/tss_template_nocomments.yaml

```diff
@@ -0,0 +1,53 @@
+sub_exp_tss: False
```

Comment:
The name of this file makes me wonder if it should be included vs. tts_template

Reply:
my rationale was maybe that the tss_template.yaml is really long, and this one can be used for people who don't want to wade through lots of comments? But now that I think twice about it, I'm wondering if that is overkill 😂

Reply:
It's long, but I don't think it's *that* long, personally. I never use or update these files though, so I defer to those who do!

Reply:
@Rplesha Ok, I removed the extra template file :)